### PR TITLE
fix: form editor error when additionalProperties is undefined

### PR DESCRIPTION
**Pull request by topliceanurazvan**
## Fixes

- Form editor error sometimes because of `additionalProperties` being undefined

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const config = {
   url: `https://${organizationName}.github.io`,
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  //baseUrl: `/${projectName}/`,
+  // baseUrl: `/${projectName}/`,
```
Comment (mortada-codes) [inline docs/docusaurus.config.js:21]: is this related to the PR?

**Review by devcatalin — APPROVED** [review]

**Review by olensmar — APPROVED** [review]

File: docs/docusaurus.config.js (hunk 1 above)
Comment (topliceanurazvan) [reply docs/docusaurus.config.js:21]: It was just some linting problem
